static-site/src/components/nav-bar/index.jsx

```diff
@@ -120,14 +121,8 @@ class NavBar extends React.Component {
         {this.getLogo()}
         {this.getLogoType()}
         <div style={{flex: 5}}/>
-        {this.selectedClass("help") ?
-          <NavLinkInactive minified={minified}>HELP</NavLinkInactive> :
-          <NavLinkToBeHandledByGatsby minified={minified} to="/help">HELP</NavLinkToBeHandledByGatsby>
-        }
-        {this.selectedClass("docs") ?
-          <NavLinkInactive minified={minified}>DOCS</NavLinkInactive> :
-          <NavLinkToBeHandledByGatsby minified={minified} to="/docs">DOCS</NavLinkToBeHandledByGatsby>
-        }
+        <NavLinkToGoToServer minified={minified} href="https://docs.nextstrain.org/en/latest/learn/about-nextstrain.html" target="_blank" rel="noopener noreferrer">HELP <FaExternalLinkAlt/></NavLinkToGoToServer>
```

Comment:
Same comment as with the auspice nav-bar - these are still considered internal links and thus shouldn't have a `<FaExternalLinkAlt/>` icon.

Reply:
Same question as with auspice nav-bar: We still want to use NavLinkToGoToServer instead of NavLinkToBeHandledByGatsby to be
> consistent with our plan to remove all "docs" and "help" content from the Gatsby section of the site.

correct?

Reply:
Yup - `<NavLinkToGoToServer>` is simply a styled `<a>` element, whereas (from memory) `<NavLinkToBeHandledByGatsby>` acts via an event handler which causes problems when you want to go to a non-Gatsby page